src/Plugin.php

```diff
@@ -2553,23 +2553,26 @@ public static function getSpecificValueToDisplay($field, $values, array $options
                 if (in_array($state, [self::ACTIVATED, self::NOTUPDATED, self::TOBECONFIGURED, self::NOTACTIVATED], true)) {
                    // Uninstall button for installed plugins
                     if (function_exists("plugin_" . $directory . "_uninstall")) {
-                        $output .= TemplateRenderer::getInstance()->render('components/plugin_uninstall_modal.html.twig', [
-                            'plugin_name' => $plugin->getField('name'),
+                        $output .= TemplateRenderer::getInstance()->render('components/danger_modal.html.twig', [
                             'modal_id' => 'uninstallModal' . $plugin->getField('directory'),
                             'open_btn' => '<a class="pointer"><span class="fas fa-fw fa-folder-minus fa-2x me-1"
                                                   data-bs-toggle="modal"
                                                   data-bs-target="#uninstallModal' . $plugin->getField('directory') . '"
                                                   title="' . __s("Uninstall") . '">
                                                   <span class="sr-only">' . __s("Uninstall") . '</span>
                                               </span></a>',
-                            'uninstall_btn' => Html::getSimpleForm(
+                            'confirm_btn' => Html::getSimpleForm(
                                 static::getFormURL(),
                                 ['action' => 'uninstall'],
                                 _x('button', 'Uninstall'),
                                 ['id' => $ID],
                                 '',
                                 'class="btn btn-danger w-100"'
                             ),
+                            'content' => sprintf(
+                                __('By uninstalling the "%s" plugin you will lose all the data of the plugin.'),
+                                $plugin->getField('name')
```

Comment:
`content` will not be escaped anymore due to the usage of the `|raw` filter
```suggestion
                                __s('By uninstalling the "%s" plugin you will lose all the data of the plugin.'),
                                htmlspecialchars($plugin->getField('name'))
```